packages/web5/lib/src/crypto/key_manager.dart

```diff
@@ -24,7 +24,7 @@ abstract interface class KeyManager {
   /// Generates and securely stores a private key based on the provided
   /// algorithm. Returns a unique alias that can be utilized to reference the
   /// generated key for future operations.
-  Future<String> generatePrivateKey(AlgorithmId algId);
+  Future<Jwk> generatePrivateKey(AlgorithmId algId);
```

Comment:
This returns the JWK but doc says it returns alias? If it is to return Jwk then should mention how to get the alias like: 

`jwk.computeThumbprint()`

Reply:
(looking at code thumbprint == alias for key manager, assume that is ok?). Minor thing

Reply:
Yeah honestly I'm not sure why the key manager originally returned the publickey thumbprint. It seems like less steps if we return the JWK instead of having to recreate the JWK for the generated private key.

Another solution is to utilize the flow I used for the DidWeb.create where we just generate the private key outside of the key manager and just import it to the key manager. This means the KeyManager.generatePrivateKey would still return the public key thumbprint.

Any thoughts @mistermoe ?

Reply:
@StonePack I think the idea of having it return an alias and then some way to export comes from:  https://github.com/TBD54566975/web5-kt/blob/main/crypto/src/main/kotlin/web5/sdk/crypto/KeyManager.kt

Which mens it can support some regimes where the key never leaves the hardware so you have no means to get at it anyway, if I understand correctly

Reply:
I updated the key generation and storage in the key manager to match the way it's being done in the web did PR code.